src/main/java/de/hpi/bpt/argos/api/entity/EntityEndpointImpl.java

```diff
@@ -0,0 +1,332 @@
+            jsonEventType.addProperty("Name", eventType.getName());
+            jsonEventType.addProperty("NumberOfEvents",
+                    PersistenceAdapterImpl.getInstance().getEventCountOfEventType(eventType.getId()));
+            jsonEventType.addProperty("TimestampAttributeId", eventType.getTimeStampAttributeId());
+
+            return jsonEventType;
+        } catch (Exception e) {
+            LoggerUtilImpl.getInstance().error(logger, "cannot parse event type", e);
+            return new JsonObject();
+        }
+    }
+
+    /**
+     * This method returns event types as a JsonObject.
+     * @param eventTypes - the event types
+     * @return - a json representation of the event types
+     */
+    private JsonObject getEventTypesJson(List<EventType> eventTypes) {
+        JsonArray eventTypesJson = new JsonArray();
+        for (EventType eventType : eventTypes) {
+            eventTypesJson.add(getEventTypeJson(eventType));
+        }
+        return eventTypesJson.getAsJsonObject();
+    }
+
+    /**
+     * This method returns events as a JsonObject.
+     * @param events - the events
+     * @return - a json representation of the events
+     */
+    private JsonObject getEventsJson(List<Event> events) {
+        JsonArray eventsJson = new JsonArray();
+        for (Event event : events) {
+            List<Attribute> attributes = PersistenceAdapterImpl.getInstance().getAttributes(event.getId());
+            JsonArray jsonAttributes = getAttributesJson(attributes);
+            eventsJson.add(jsonAttributes);
+        }
+        return eventsJson.getAsJsonObject();
+    }
+
+    /**
+     * This method returns attributes as a JsonObject.
+     * @param attributes - the attributes
+     * @return - a json representation of the attributes
+     */
+    private JsonArray getAttributesJson(List<Attribute> attributes) {
+        JsonArray jsonAttributes = new JsonArray();
+        for (Attribute attribute : attributes) {
+            JsonObject jsonAttribute = new JsonObject();
+            TypeAttribute typeAttribute = PersistenceAdapterImpl.getInstance().getTypeAttribute(attribute.getTypeAttributeId());
+            jsonAttribute.addProperty("Name", typeAttribute.getName());
+            jsonAttribute.addProperty("Value", attribute.getValue());
+            jsonAttributes.add(jsonAttribute);
+        }
+        return jsonAttributes;
+    }
+
+    /**
+     * This method returns a query to query if there exists an event query for the given entity and event type.
+     * @param entityId id of the entity
+     * @param eventTypeId id of the vent type
+     * @return query string to find out if event query exists
+     */
+    private String getExistsEventQuery(long entityId, long eventTypeId) {
+        String eventTableName = "Events";
+        return String.format(
+                "SELECT CASE WHEN EXISTS (" +
+                        "SELECT *" +
+                        "FROM %1$s" +

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') '+' should be on a new line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AOperator_Wrap_BP2016)